### PR TITLE
ci: remove gulp from the project

### LINKED ISSUE #113: Remove gulp from project
I suggest to remove `gulp` from the project and change command for run linting from gulp to script in `package.json`

### PR DESCRIPTION
**Pull request by romanrostislavovich**
Removed `gulp` and all his dependencies from the project and command for run linting moved to script in `package.json`

Closes #113

### CODE REVIEW

**Comment by xferra** [comment]
Please doublecheck runtime deps, seems like some of them are dev deps.

**Comment by romanrostislavovich** [comment]
`jsonschema` - usage in `configValidation` 
`eslint-config-google` and `eslint` - needs for linting

Other removed packages were used in `gulp`

**Comment by romanrostislavovich** [comment]
Sorry, I understood your comment wrong. I fixed it

File: package.json [hunk 1]
```diff
@@ -10,21 +10,21 @@
...
-        "gulp-eslint": "^5.0.0",
-        "gulp-hub": "^0.8.0",
-        "jsonschema": "^1.2.4"
+        "eslint-config-google": "^0.9.1"
     },
     "dependencies": {
-        "chai": "^4.1.2",
-        "commander": "^2.16.0",
-        "mocha": "^5.2.0"
+        "fs": "0.0.1-security",
```
Comment (xferra) [inline package.json]: I suppose this is not mandatory.

Comment (romanrostislavovich) [reply package.json]: I fix it

**Review by ghost — APPROVED** [review]

**Pull request merged**